app/controllers/api/base_controller/renderer.rb

```diff
@@ -413,7 +425,7 @@ def render_options(resource, data = {})
                                                               collection.virtual_attribute_names),
               :virtual_attributes => options_attribute_list(collection.virtual_attribute_names),
               :relationships      => (collection.reflections.keys |
-                                       collection.virtual_reflections.keys.collect(&:to_s)).sort
+                collection.virtual_reflections.keys.collect(&:to_s)).sort
```

Comment:
was this for Rubocop too ?  looks odd, can collection on 428 align with collection on 427 while keeping Rubocop happy ?